charts/seqr/templates/update-all-reference-data-job.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ $.Chart.Name }}-update-all-reference-data
+  labels:
+    name: {{ $.Chart.Name }}-update-all-reference-data
+    {{- include "seqr.labels" $ | nindent 4 }}
+  annotations:
+    "helm.sh/hook": post-install, post-upgrade
```

Comment:
I do not think this should be done `post-upgrade`, just `post-install`. I think longer term I will add a better job for just updating the updated resources as a cron job/ pot update task